src/cr/cube/cubepart.py

```diff
@@ -570,10 +573,10 @@ def scale_median_column(self):
         The median is calculated using the standard algebra applied to the numeric
         values repeated for each related counts value
```

Comment:
this could get us into trouble with weights — how does repeat deal with floats? 
weighted median is a kind of weird case to begin with, but this method isn't fully robust to weighted counts.